docs/src/main/tut/typeclasses/lawtesting.md

```diff
@@ -0,0 +1,146 @@
+# Law testing
+
+Laws are an important part of cats.
```

Comment:
Laws are explained [here](https://typelevel.org/cats/typeclasses.html), maybe make that section a linkable anchor and link from here?

Reply:
Markdown embeds `id`s so, we don't even need an anchor 🎉. Great suggestion, will do!